New log file opened at Mon Dec 07 13:11:23 EST 2015 by 237616@SkyFort
Taking next connection to: 49203
New client attempting to connect on port: 49204-49203
  Address = /192.168.56.1:49203
This client on port 49204 has uuid 346b8ca8-825a-498b-90fd-e93dc7872da5 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-346b8ca8-825a-498b-90fd-e93dc7872da5
Taking next connection to: 49203
New client attempting to connect on port: 49222-49203
  Address = /192.168.56.1:49203
Handshake exception on socket: 49222
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 49203
New client attempting to connect on port: 49223-49203
  Address = /192.168.56.1:49203
This client on port 49223 has uuid d48396bd-b211-4572-bf9f-92d959c1cc75 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-346b8ca8-825a-498b-90fd-e93dc7872da5, play-fork-run-d48396bd-b211-4572-bf9f-92d959c1cc75
Taking next connection to: 49203
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 49203
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 49203
New client attempting to connect on port: 49520-49203
  Address = /192.168.56.1:49203
Handshake exception on socket: 49520
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 49203
New client attempting to connect on port: 49521-49203
  Address = /192.168.56.1:49203
This client on port 49521 has uuid 2b138109-03a3-409d-810a-e77ebe7e8993 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-346b8ca8-825a-498b-90fd-e93dc7872da5, play-fork-run-2b138109-03a3-409d-810a-e77ebe7e8993
Taking next connection to: 49203
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 49203
